### ISSUE TITLE
Tailwind LSP support in Ruby files for Phlex Components

### ISSUE THREAD

**Issue by bradgessler** [zed-industries/community#2241]
### Check for existing issues

- [X] Completed

### Describe the feature

Similar to https://github.com/zed-industries/community/issues/2153 and https://github.com/zed-industries/community/issues/2057, Tailwind LSP auto-complete would be useful in Ruby (.rb) code for authoring Phlex components.

### If applicable, add mockups / screenshots to help present your vision of the feature

I have this working in Sublime Text at https://gist.github.com/bradgessler/1907d07593918fb5d2f27911bccc75f4

Here's what it looks like in .rb files when decorating the `class:` attribute with a Tailwind class:

<img width="806" alt="Screenshot 2023-11-07 at 8 22 01 AM" src="https://github.com/zed-industries/community/assets/4628/369a3fb1-e1f0-4200-af06-b0d90aa8d759">

**Comment by joeldrapper** [comment]
I agree this would be a kiler feature. One way to get there might be to support language server configuration so the community can build support for things like this. See https://github.com/zed-industries/community/issues/1968

I would be a lot more motivated to build a Phlex extension for `ruby-lsp` if I could configure Zed to use it.
